src/modules/hunting/cvehunter.py

```diff
@@ -32,7 +32,9 @@ def __init__(self, evidence):
 # Passive Hunter
 @handler.subscribe(ApiServer)
 class IsVulnerableToCVEAttack(Hunter):
-    """ Node is running a Kubernetes version vulnerable to critical CVE-2018-1002105 """
+    """API server version end point and CVE-2018-1002105 hunter
+    Checks if the API server version end point is accessible or node is running a Kubernetes version which vulnerable to critical CVE-2018-1002105
```

Comment:
This is inaccurate / ambiguous - reads as though the result may be showing simply that the API server end point is accessible. Why did this message need to change? 

(Also, grammar-wise, it should be "which is vulnerable").